polars/polars-lazy/polars-plan/src/dsl/arithmetic.rs

```diff
@@ -127,12 +127,30 @@ impl Expr {
         self.map_private(FunctionExpr::Trigonometry(TrigonometricFunction::ArcCosh))
     }
 
+    /// Compute the inverse hyperbolic sine of the given expression
+    #[cfg(feature = "trigonometry")]
+    pub fn arcsinh(self) -> Self {
+        self.map_private(FunctionExpr::Trigonometry(TrigonometricFunction::ArcSinh))
+    }
+
     /// Compute the inverse hyperbolic tangent of the given expression
     #[cfg(feature = "trigonometry")]
     pub fn arctanh(self) -> Self {
         self.map_private(FunctionExpr::Trigonometry(TrigonometricFunction::ArcTanh))
     }
 
+    /// Convert from radians to degrees
```

Comment:
Can you also expose the new functions to the python API?

They should also get an entry in the python docs.

Reply:
I added them to `general.rs`, please let me know if that looks good. I can add unit tests from the different entry point aswell if requested.